external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -1491,4 +1494,38 @@ class AvroSuite extends QueryTest with SharedSQLContext with SQLTestUtils {
       |}
     """.stripMargin)
   }
+
+  test("SPARK-27798: from_avro produces same value when converted to local relation") {
```

Comment:
Please move this test case to `AvroFunctionsSuite`.

Reply:
Moved. Thanks.